src/main/java/com/exercise/sfe/exception/PersistenceException.java

```diff
@@ -0,0 +1,14 @@
+package com.exercise.sfe.exception;
+
+import lombok.Getter;
+import lombok.RequiredArgsConstructor;
+import lombok.Setter;
+
+@Getter
+@Setter
+@RequiredArgsConstructor
+public class PersistenceException extends RuntimeException {
+
+  private final int code;
+  private final transient String body;
```

Comment:
Maybe it would be better to name it message? and why it is transcient? do you need it?